pkgs/by-name/pr/prettier/package.nix

```
@@ -0,0 +1,68 @@
+{
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
```

Perhaps we should use what's recommended in the manual by default?

```suggestion
  makeBinaryWrapper,
```